ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -1,5 +1,74 @@
 [
   {
+    "id": "java-centos-mysql",
+    "creator": "ide",
+    "name": "Java-MySQL CentOS",
+    "description": "Multi-machine environment with Java CentOS Stack and MySQL database",
+    "scope": "general",
+    "tags": [
+      "Java 1.8, Tomcat 8, MySQL 5.7"
+    ],
+    "components": [
+      {
+        "name": "JDK",
+        "version": "1.8.0_45"
+      },
+      {
+        "name": "Maven",
+        "version": "3.2.2"
+      },
+      {
+        "name": "Tomcat",
+        "version": "8.0.24"
+      }
+    ],
+    "source": {
+      "type": "image",
+      "origin": "rhche/centos_jdk8"
+    },
+    "workspaceConfig": {
+      "environments": {
+        "default": {
+          "machines": {
+            "dev-machine": {
+              "agents": [
+                "org.eclipse.che.terminal", "org.eclipse.che.ws-agent", "com.redhat.bayesian.lsp"
```

Comment:
That's a good point @sunix. I think @davidfestal did it for consistency since other stacks use the same agent here. But I understand that's an issue for you because you are in the middle of a rebasing. I think we have 2 choices:
- @davidfestal remove `com.redhat.bayesian.lsp` and we merge it right now
- @davidfestal change the PR to be merge on `master` branch instead of `openshift-connector`

@sunix are these 2 options ok for you?
@davidfestal which of these 2 options is better for you?